corehq/apps/app_manager/suite_xml/post_process/workflow.py

```diff
@@ -435,14 +420,9 @@ def _add_stack_children_for_target(self, stack_frames, target_module, source_for
         :param source_form_datums: List of datums from the source form.
         """
         ids_on_stack = stack_frames.ids_on_stack
-        command = None
-        if len(target_module.forms):
-            command = id_strings.form_command(target_module.get_form(0))
-        elif target_module.case_list and target_module.case_list.show:
-            command = id_strings.case_list_command(target_module)
-
-        if command:
-            target_frame_children = self.helper.get_frame_children(command, target_module, module_only=True)
+
+        if target_module.forms or (target_module.case_list and target_module.case_list.show):
```

Comment:
I'm not convinced that this `if` statement is necessary, but I didn't want to risk behavior changes

Reply:
ooh good point, glad I didn't just ignore that. Thanks.